example/dapps/helpers/userAgent.ts

```diff
@@ -0,0 +1,8 @@
+const userAgentIsMobile = (): boolean => {
```

Comment:
@walletconnect/browser-utils exports a function `isMobile`, probably a good idea to use that instead